ebssurrogate/scripts/surrogate-bootstrap.sh

```diff
@@ -259,6 +259,9 @@ function clean_system {
 	mkdir /mnt/var/log/wal-g
 	touch /mnt/var/log/wal-g/{backup-push.log,backup-fetch.log,wal-push.log,wal-fetch.log,pitr.log}
 
+	#Creatre Sysstat directory for SAR
+	mkdir /mnt/var/log/sysstat
```

Comment:
```suggestion
	mkdir -p /mnt/var/log/sysstat
```
Just incase anything in the tree doesn't exist